Documentation/Introduction/Index.rst

```diff
@@ -13,84 +13,80 @@ Introduction
-At the same time the process is not overly complicated.
+In building the sitepackage as an extension, all relevant files are stored in
+one place and changes can easily be tracked in a version control system
+such as Git. The site package approach is not the only way of creating TYPO3
+websites but it is flexible and professional and not overly complicated.
 
 Dependency Management
 ---------------------
-Another important benefit of a TYPO3 extension is the fact that dependencies to
-other extensions and/or the TYPO3 version can be defined. This makes
-deployments easier and more fail-safe. This feature is called "Dependency
-Management". Most TYPO3 sites require a number of extensions. This could be
-"News" or "Powermail" for example. By building a sitepackage
-extension, which may contain global configuration for these add-ons, the
-dependencies can be defined. When the sitepackage extension is installed in an
-empty TYPO3 instance, all dependent extensions are downloaded from the `TYPO3
-Extension Repository <https://extensions.typo3.org>`__ and installed
-automatically.
+TYPO3 extensions allow dependencies to other extensions and/or the TYPO3 version
+to be defined. This is called "Dependency Management" and makes deployment easy
+and fail-safe. Most TYPO3 sites are dependent on a number of extensions. Some
+examples are "News" or "Powermail". A sitepackage extension which contains
+global configuration variables for these extensions will define the dependencies
+for you. When the sitepackage extension is installed in an
+empty TYPO3 instance, all dependent extensions are automatically downloaded from
+the `TYPO3 Extension Repository <https://extensions.typo3.org>`__ and installed.
 
 Clean Separation from the Userspace
 -----------------------------------
-Without using an extension, template files are often stored in the
-:file:`fileadmin/` directory of a TYPO3 instance. As a matter of fact, files in
-this directory are indexed by TYPO3's File Abstraction Layer (FAL). This
-results in irrelevant and avoidable records in the database. From a logical
-perspective, the :file:`fileadmin/` area is the "userspace". This area is meant to
-belong to editors. Even if access permissions restrict editors from accessing
-or manipulating files in :file:`fileadmin/`, site configuration components should
-not be stored in the userspace by all means.
+In a TYPO3 installation that doesn't use extensions, template files are often
+stored in the :file:`fileadmin/` directory. Files in
+this directory are indexed by TYPO3's File Abstraction Layer (FAL) resulting in
+possibly irrelevant records in the database. To avoid this the :file:`fileadmin/`
+area should be seen as a "userspace" which is only available for editors to
+use. Even if access permissions restrict editors from accessing or manipulating
+files in :file:`fileadmin/`, site configuration components should
+still not be stored in the userspace.
```

Comment:
In my understanding "still" is the wrong word ("immer noch").

Reply:
Here it means "trotzdem"